File: start.ts
const LogisticInfo = (products: Product[]) => {
    console.log(products)
}

class Product {
    id: string;
    name: string;
    slas: string[];
}

const products = [
    {
      id: "1234",
      name: "Camisa polo",
      slas: ["Entrega Agendada", "Entrega Correios"]
    },
    {
      id: "567",
      name: "Calça jeans",
      slas: ["Entrega Correios", "Retirada em loja"]
    },
    {
      id: "890",
      name: "Vestido",
      slas: ["Retirada em loja"]
    }
  ]
  

LogisticInfo(products)

File: start.py
def LogisticInfo(products):
    
    print(str(products))


class Product:
    def __init__(self, id, name, slas):
        self.id = id
        self.name = name
        self.slas = slas

def main():
    products = []
    products.append(Product("1234", "Camisa polo", ["Entrega Agendada", "Entrega Correios"]))
    products.append(Product("567","Calça jeans",["Entrega Correios", "Retirada em loja"]))
    products.append(Product("890","Vestido",["Retirada em loja"]))

    LogisticInfo(products)

if __name__== "__main__":
  main()